modules/statics/src/base.ts

```diff
@@ -420,11 +420,13 @@ export enum UnderlyingAsset {
   IVY = 'ivy',
   JBC = 'jbc',
   JFIN = 'jfin',
+  JOE = 'joe',
```

Comment:
why are we adding these to ERC-20? Please add them add at the bottom (look at 703-726 as example). Add them like avaxc-bridge:JOE

Reply:
Done!